src/components/WebexAvatar/WebexAvatar.scss

```diff
@@ -0,0 +1,37 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-avatar {
+  display: flex;
+  align-items: center;
+
+  font-size: .875rem;
+  line-height: 1.25rem;
+
+  width: 2.5rem;
+  height: 2.5rem;
+
+  position: relative;
+
+  img {
+    border-radius: 50%;
+  }
+
+  .status-icon {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    left: 62.5%;
+    right: -12.5%;
+    top: 56.25%;
+    bottom: -6.25%;
+
+    border-radius: 50%;
+
+    background: var(--wxc-icon--background-color);
+
+    .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-icon {
```

Comment:
please send an optional class name to the Icon component for that, instead of targeting its base class